### PR TITLE
Medium NFSD Interdictor - Wendigo (New)

### PR DESCRIPTION
**Pull request by MagnusCrowe**
## About the PR
This PR adds:
- A new medium ship

**The Wendigo**
Ship Size: Medium
Recommended Crew: 2-3 operators + 1 boarding party when interdicting a vessel
Cost: 50000 spesos
Power Gen Type: Uranium
Expeditions: None
IFF console: Yes
Available Roles: None
Purchasable by: Sergeant and up

The Wendigo is a light medium interdictor vessel.  It is equipped with the powerful M_EMP.   The M_EMP is a device that projects a huge EMP field in all directions, likely to fully disable at least one ship entirely.  Ships that are not equipped with Nanotrasen EMP shielding technology will find all their electronic systems depleted.  This is the same system installed on the marauder.

The recommended crew includes a pilot and a weapons officer.  There is a seat on the bridge for a communications officer as well if the need arises.  The ship has a couple options for boarding parties to stage.  There is a modest cargo/docking bay that deputies can use to board a EMP disabled ship.  There are also nacelle catwalks where deputies with jetpack and hardsuit can stage for a multi-point entry boarding assault on disabled ships.

This ship is intended to be used as a penultimate resort for particularly belligerent and evasive criminals.  It will stop an evasive vessel without harming it permanently.  

Disclaimer: The M_EMP system requires authorization from the Sheriff to operate.  The possibility of hitting innocent ships is very real and due care needs to be taken in the use of the M_EMP.

## Why / Balance

I feel that NFSD needs a more focused interdiction vessel.  One that can be operated by a smaller crew than the Marauder.

## How to test

As you would with any ship.

## Media

image
image
image

## Requirements

- [x ] I have read and I am following the [Pull Request Guidelines](https://docs.spacestation14.com/en/general-development/codebase-info/pull-request-guidelines.html). I understand that not doing so may get my pr closed at maintainer’s discretion
- [x ] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase

## Breaking changes

None

**Changelog**

:cl: MagnusCrowe
- add: Added the Wendigo light medium interdictor!

### CODE REVIEW

**Comment by MagnusCrowe** [comment]
Downgraded the radar so it doesn't encroach on the Prowler's niche.

**Comment by MagnusCrowe** [comment]
> when do pirates get an M_EMP i wonder

Great question.

**Review by arimah — CHANGES_REQUESTED** [review]
I fucking love the overall profile of this ship, the functionality is offers is great, it flies like a dream, it looks mean and cool as shit. I like it a lot. This is a great example of an NFSD ship that is focused, offers something good, and isn't powergamey as hell.

The following are mostly nitpicks, as the ship is otherwise flawless.

The ship appraises at ~29,650, but costs 50,000. I'm not sure what's changed there, but that seems like too steep of a markup, especially for an NFSD ship.

image
1. Consider colouring all of these pipes, like we've been doing on other ships.
2. Also, I'm not _hugely_ fond of having the generator in the middle of the room like that. I get it's probably to keep the APC accessible, but something about it it feels off to me. It's not a blocker.

image
1. No amount of poster shenaniganry can cover up your crimes here, mister. The fact that _every single_ transition from plastitanium to rwall has a conspicuously placed poster makes it look super weird to me. I can't think of any better way to make this awkward wall transition work, but still, aaaa.
2. Why do two of these airlocks have firelocks under them, while the other two have thin firelocks in front of them? 🤔 It's not a blocker, I just don't really get it.

image
Mysterious MV cables?

Piping is clean as hell, nice. 👌

**Comment by PeccNeck** [comment]
> The ship appraises at ~29,650, but costs 50,000. I'm not sure what's changed there, but that seems like too steep of a markup, especially for an NFSD ship.

I feel like chiming in here and say that it's not the only NFSD ship with an egregious markup, nor is it the worst; at a ~69% markup (nice), it's comparable to the Interceptor with its ~51% markup. There's also the Hospitaller and Inquisitor, both of which are marked up by ~197% (nearly 3x their base value!).

Still pretty bad though, especially considering that those are all Smalls, and at least the latter two run off of APUs.

**Comment by MagnusCrowe** [comment]
It's markup is to keep it on the same price tier as the prowler.  I figured it wasn't a huge deal since most of the time the thing will be purchased with a voucher anyway.  I actually want to encourage it to be bought with a voucher so I left it.

**Comment by MagnusCrowe** [comment]
The MV cables are to electrify the windoors.

Firelocks work perfectly fine on transparent doors but not fine on solid doors.  Solid doors get the baby firelock infront for that reason.

**Comment by MagnusCrowe** [comment]
As for the wall sins, I just prefer that the internal walls of a ship be weaker than the hull.  I wish we had a more cooperative texture for plastitanium but it is what it is.  If necessary we CAN change the internal walls to plastitanium but I feel like that would lower the chance of destructive shenanigans inside the ship.

I'll let you decide on that.

**Comment by arimah** [comment]
>  I actually want to encourage it to be bought with a voucher so I left it.

50k is by no means a prohibitive amount once you've played for a few hours. I don't know if it's expensive _enough_ to encourage voucher use. Given that, you probably shouldn't lose 20k on it if you decide to forgo the voucher. I understand your reasoning, but don't agree with it...

If we decide to start allowing big markups on NFSD ships, we should amend the ship submission guidelines and the shipyard article.

> The MV cables are to electrify the windoors.

Since when can you do that?! The windoors don't even have MV connectors. 🤔

**Comment by MagnusCrowe** [comment]
> > I actually want to encourage it to be bought with a voucher so I left it.
> 
> 50k is by no means a prohibitive amount once you've played for a few hours. I don't know if it's expensive _enough_ to encourage voucher use. Given that, you probably shouldn't lose 20k on it if you decide to forgo the voucher. I understand your reasoning, but don't agree with it...
> 
> If we decide to start allowing big markups on NFSD ships, we should amend the ship submission guidelines and the shipyard article.
> 
> > The MV cables are to electrify the windoors.
> 
> Since when can you do that?! The windoors don't even have MV connectors. 🤔

Changing the price is fine with me.

**Comment by MagnusCrowe** [comment]
image
I fixed the price, replaced the inner walls, and removed the extra mv cable.

File: Resources/Maps/_NF/Shuttles/Nfsd/wendigo.yml [hunk 1]
```diff
@@ -768,6 +768,20 @@ entities:
...
+    - type: Transform
+      rot: -28.27433388230815 rad
+      pos: -1.3030288,-3.779652
+      parent: 1
+  - uid: 472
+    components:
+    - type: Transform
+      rot: -21.991148575128555 rad
+      pos: 2.3636386,-3.758803
+      parent: 1
```
Comment (arimah) [inline Resources/Maps/_NF/Shuttles/Nfsd/wendigo.yml:784]: I am enormously intrigued by the presence of these two pens. Contraband pens on an NFSD ship?! How did this happen? 😄

Comment (MagnusCrowe) [reply Resources/Maps/_NF/Shuttles/Nfsd/wendigo.yml:784]: Contraband pens woops

Comment (MagnusCrowe) [reply Resources/Maps/_NF/Shuttles/Nfsd/wendigo.yml:784]: what the hell is an advanced pen?

Comment (arimah) [reply Resources/Maps/_NF/Shuttles/Nfsd/wendigo.yml:784]: It's the base prototype for e-daggers, explosive pens and hypopens. It should probably be abstract. Blame upstream.

**Review by arimah — APPROVED** [review]
This is a good ship.

**Pull request merged**